rpcs3/Emu/Cell/Modules/cellAdec.cpp

```diff
@@ -554,14 +530,7 @@ class AudioDecoder : public ppu_thread
 						break;
 					}
 
-					last_frame = av_read_frame(fmt, &au) < 0;
-					if (last_frame)
-					{
-						//break;
-						av_free(au.data);
-						au.data = NULL;
-						au.size = 0;
-					}
+					av_read_frame(fmt, packet);
```

Comment:
Why is the result not checked?

Reply:
Previous code was destructor's job, so it shouldn't affect anything.